packages/jupyterlab-collaborative-chat/src/widget.tsx

```diff
@@ -283,6 +309,26 @@ class ChatSection extends PanelWithToolbar {
     return (this.widgets[0] as ChatWidget).model;
   }
 
+  /**
+   * Dispose of the resources held by the widget.
+   */
+  dispose(): void {
+    this.model.unreadChanged?.disconnect(this._unreadChanged);
+    super.dispose();
+  }
+
+  /**
+   * Change the title when messages are unread.
+   *
+   * TODO: fix it upstream in @jupyterlab/ui-components.
+   * Updating the title create a new Title widget, but does not attach again the
+   * toolbar. The toolbar is attached only when the title widget is attached the first
+   * time.
+   */
+  private _unreadChanged = (_: IChatModel, unread: number[]) => {
+    // this.title.label = `${unread.length ? '* ' : ''}${this._name}`;
```

Comment:
Should we remove this, as you implemented it in CSS?

Reply:
Thanks for reviewing it @martinRenou.

The CSS you saw is for the main area tab.
This part is for the sidebar title, related to https://github.com/jupyterlab/jupyterlab/pull/16390.

As soon as the PR above is released, we'll be able change the minimum version of jupyterlab and uncomment this line.